Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1799,6 +1799,28 @@ For more information, ensure that you have the fix `Pull Request <https://github
 Known Issues
 ############
 
+Connection Collisions When a Service Endpoint Is Accessed via Multiple VIPs
```

Comment:
What's your gut feeling for a client that connects to a backend twice:
* once via a VIP
* once directly (via public reachable PodIP)

I would expect similar conflicts - replies for the "direct" connection are unexpectedly revDNATed ?